### PR TITLE
🎉  Starting the home page overview #4

### PR DESCRIPTION
**Pull request by jelizasilatei**
# Description

Build an overview homepage.

Fixes #4

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jelizasilatei** [comment]
Kindly review my pull request, I worked on this with Victor and this is his github account https://github.com/Vickynjoroge.

File: index.html [hunk 1]
```diff
@@ -14,6 +14,30 @@
 
 <body>
   <div>Welcome</div>
-</body>
 
+  <!-- Overview Container -->
+  <div class="overview-container">
+    <!-- Overview Image Section -->
+    <div class="overview-image-section">
+      <img id="overview-image" src="src/assests/gettyimages-98473330-2048x2048@2x.png" alt="Your Image">
```
Comment (JAPHETHNYARANGA) [inline index.html:20]: provide better alternative name

File: src/css/style.css [hunk 2]
```diff
@@ -1,5 +1,137 @@
...
+.overview-container {
+    background-color: #DBC2F5; /* Background color for the overview container */
+    width: 100%; /* Make it 100% wide */
+}
+
+/* Style for the overview image section */
+.overview-image-section {
+    width: 100%; /* Make it 100% wide to ensure responsiveness */
+    justify-content: center;
+}
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: if the width of th image div is 100% , where are the other text elements that need to be beside the image placed

File: src/ts/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+console.log ('hi');
```
Comment (JAPHETHNYARANGA) [inline src/ts/index.ts]: edit only required files

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Please provide web and mobile screenshot of the page

**Comment by jelizasilatei** [comment]
Screenshots for Web and Mobile page.
Web:
Web screenshot for home page overview section

Mobile:
Mobile screenshot for home page overview section

File: index.html (hunk 1 above)
Comment (jelizasilatei) [reply index.html:20]: `  
  <div class="homepage-overview-container">
    
    <div class="homepage-overview-image">
      Your Image
    </div>`

File: src/css/style.css (hunk 2 above)
Comment (jelizasilatei) [reply src/css/style.css]: I used a Mobile-First Responsive Design the mobile and tablet page has a width of 100% which mean the image and text elements will be vertically arrange.
The text image comes below the image.
When it is on a desktop screen the image floats to the left and text elements float to the right

File: src/ts/index.ts (hunk 3 above)
Comment (jelizasilatei) [reply src/ts/index.ts]: I have deleted this line of code.

**Review by JAPHETHNYARANGA — APPROVED** [review]

**Comment by JAPHETHNYARANGA** [comment]
Good work, but next time dont delete code already in the specific file you are editing

**Comment by jelizasilatei** [comment]
Okay

**Pull request closed without merging**